tft_bot/config.py

```diff
@@ -109,3 +109,25 @@ def get_override_install_location() -> str | None:
 
     """
     return _SELF.get("override_install_location") or None
+
+
+def get_wanted_traits() -> list[str]:
+    """
+    Get the list of traits we attempt to purchase.
+
+    Returns:
+        A list of traits we look for.
+
+    """
+    return _SELF.get("wanted_traits", ["duelist", "brawler"])
+
+
+def purchase_traits_in_prioritized_order() -> bool:
+    """
+    Get if we should purchase traits in prioritized order.
+
+    Returns:
+        True if we should only buy a trait if the trait before it has been found, False if not.
+
+    """
+    return _SELF.get("purchase_traits_in_prioritized_order", True)
```

Comment:
While this technically should default to `True`, I think we should play around with having it `False` for a while ourselves and see if that increases the win rate (top 4), since logically that'd make a bit more sense to me, I think? 🤔 

To be completely honest, I'm not sure a scenario where we'd actually want this if my theory is right, but since it's relatively trivial to include, I'm fine with that. This was kept because the original bot pre-fork had that logic implemented and it worked "good enough" so I'd been lazy in updating the evaluation 😛

Reply:
It skews your team comp towards the list order, so it's preferred if you play duelist brawler. I've switched to playing purely duelist, so I can't comment on the effectiveness, but I'll work on an LCU method to get (and print out) the average win rate of the last 20 games or so, so we can idle the bot and tell if a comp/setting benefits or hurts us :)